src/models/transfer/facade.js

```diff
@@ -290,6 +296,69 @@ const saveTransferFulfilled = async (transferId, payload, isCommit = true, state
   }
 }
 
+const saveTransferAborted = async (transferId, payload, eventAction) => {
```

Comment:
Please add JS Docs to this new function.

Reply:
ok